acct.bionimbus.org/etlMapping.yaml

```diff
@@ -137,7 +137,7 @@ mappings:
           - name: data_category
             src: data_type
             fn: set
-  - name: acct_file
+  - name: acct.bionimbus.org_file
```

Comment:
Why did this row change?

Reply:
This is working by design.
Our automation names the Elastic Search index according to the web address / hostname of the target environment.
This should be harmless.